client/app/vps/vps-detail.controller.js

```diff
@@ -17,6 +18,7 @@ class VpsDetailCtrl {
                 serviceName: this.serviceName
             }
         });
+        this.VpsTaskService.subscribe(this.serviceName);
```

Comment:
is-it normal to not have any unsubscribe?

Reply:
I agree this is weird.  You should have an unsubscribe function.